src/Common/src/Interop/Unix/System.Security.Cryptography.Native/Interop.OpenSsl.cs

```diff
@@ -99,6 +100,19 @@ internal static SafeSslHandle AllocateSslContext(SslProtocols protocols, SafeX50
                     context.Dispose();
                     throw CreateSslException(SR.net_allocate_ssl_context_failed);
                 }
+
+                if (certHandle != null && !certHandle.IsInvalid)
+                {
+                    using (X509Certificate2 cert = new X509Certificate2(certHandle.DangerousGetHandle()))
+                    {
+                        X509Chain chain = TLSCertificateExtensions.BuildNewChain(cert, false);
```

Comment:
Nit: could you name the Boolean argument, so that it's clear to a reader what "false" means here?